### DISCUSSION TITLE
Overwrite options for Choices.JS

### DISCUSSION THREAD

**Discussion by dejury** [filamentphp/filament#4939] [Help]
Is it somehow possible to overwrite certain settings for Choices.JS? 
For example i want to have `position: top` to make sure the options are rendered above the input.

See https://github.com/Choices-js/Choices#configuration-options

I dived a bit in the code, but it seems it isn't possible out of the box.

**Comment by danharrin** [comment]
Maybe you could just introduce a new `position()` method for the Select object?

**Reply by dejury** [reply]
I can do that, need a little help on how to test and build the javascript changes?

**Reply by danharrin** [reply]
You can run `npm run build` to recompile the JS and test it with your local Composer repo
